src/constants.js

```diff
@@ -3,97 +3,202 @@ export const MODAL_WINDOW = {
+    description: `Parasite (Korean: 기생충; RR: Gisaengchung) is a 2019 South Korean black comedy thriller film directed by 
+    Bong Joon-ho, who also co-wrote the screenplay with Han Jin-won. It stars Song Kang-ho, Lee Sun-kyun, Cho Yeo-jeong, 
+    Choi Woo-shik, Park So-dam, Jang Hye-jin, and Lee Jung-eun and follows the members of a poor family who scheme to 
+    become employed by a wealthy family by infiltrating their household and posing as unrelated, highly qualified individuals.
+
+    Parasite premiered at the 2019 Cannes Film Festival on 21 May 2019, where it became the first South Korean film to 
+    win the Palme d'Or and the first film to win with a unanimous vote since Blue Is the Warmest Colour at the 2013 Festival. 
+    It was then released in South Korea by CJ Entertainment on 30 May 2019.
+    
+    The film received widespread critical acclaim, with praise directed towards its screenplay, Bong's direction, acting, 
+    social commentary, cinematography, editing and production values, and has featured in multiple listings of the best 
+    films of the 2010s. It has grossed over $269.4 million worldwide on a production budget of about $11 million, becoming 
+    the highest-grossing South Korean film.`,
+    cinema: CINEMA_CODE.VOKA,
     time: '15:00',
+    hour: 15,
+    emptySeats: 1,
   },
 ];
 
-export const filters = [
+export const CINEMAS = {
+  [CINEMA_CODE.GALILEO]: {
+    id: 1,
+    filterName: 'Galileo',
+    cinemaName: 'Silver Screen cinemas in "Galileo',
+    address: 'city Minsk, str. Bobryiskay, 6',
+    cinemaCode: CINEMA_CODE.GALILEO,
+  },
+  [CINEMA_CODE.VOKA]: {
+    id: 2,
+    filterName: 'VOKA CINEMA',
+    cinemaName: 'VOKA CINEMA by SilverScreen in Dana Mall',
+    address: 'сity Vitebsk, str. Mstislavca, 11',
+    cinemaCode: CINEMA_CODE.VOKA,
+  },
+};
+
+export const FILTERS = [
   {
     id: 1,
-    filter: 'Город',
+    filterName: 'cityCode',
+    method: 'equal',
+    defaultValue: 'City',
+    items: [
+      { id: 1, name: 'All cities', value: VALUE_ALL },
+      { id: 2, name: 'Minsk', value: CITY_CODE.MSK },
+      { id: 3, name: 'Vitebsk', value: CITY_CODE.VTB },
+    ],
   },
   {
     id: 2,
-    filter: 'Все кинотеатры',
+    filterName: 'cinema',
+    method: 'equal',
+    defaultValue: 'All cinemas',
+    items: [
+      { id: 1, name: 'All cinemas', value: VALUE_ALL },
+      { id: 2, name: 'Galileo', value: CINEMA_CODE.GALILEO },
+      { id: 3, name: 'VOKA CINEMA', value: CINEMA_CODE.VOKA },
+    ],
   },
   {
     id: 3,
-    filter: 'Сегодня',
+    filterName: 'dateCode',
+    method: 'equal',
+    defaultValue: 'All time',
+    items: [
+      { id: 1, name: 'All time', value: VALUE_ALL },
+      { id: 2, name: 'Day after tomorow', value: DATE_CODE.DAY_AFTER_TOMORROW },
+      { id: 3, name: 'Tomorrow', value: DATE_CODE.TOMORROW },
+    ],
   },
   {
     id: 4,
-    filter: 'Все сеансы',
+    filterName: 'hour',
+    method: 'inRange',
+    defaultValue: 'All sessions',
```

Comment:
this doesn't match any of the items

Reply:
defaultValue doesn't match any item. It is used to render when no one item is selected.